packages/code-du-travail-frontend/src/home/Partners.tsx

```diff
@@ -2,6 +2,8 @@ import { Container, Section, theme, Title } from "@socialgouv/cdtn-ui";
 import Image from "next/image";
 import React from "react";
 import styled from "styled-components";
+import servicePublic from "../../public/static/assets/img/logo_sp_hd_rvb.png";
+import ministere from "../../public/static/assets/img/ministere_logo.png";
```

Comment:
```suggestion
import servicePublic from "/static/assets/img/logo_sp_hd_rvb.png";
import ministere from "/static/assets/img/ministere_logo.png";
```

Reply:
ça ne build pas si je fais ça :
`Module not found: Can't resolve '/static/assets/img/logo_sp_hd_rvb.png'`

Reply:
Ah dommage, dans ce cas là, partons sur ce que t'as fait !